### PR TITLE
Chore/improve database

### PR DESCRIPTION
**Pull request by carlomigueldy**
Though we are moving towards web3-ish and idea of iterating into DAO-to-DAOs, table updates were necessary

- Renamed the `companies` table to `organizations`
- Add new link for drawsql schema
- Update attribute data types to use `varying character` instead of `uuid` for references to `users` table primary key

drawsql url: https://drawsql.app/hefesto/diagrams/developer-dao-job-board-schema#

schema

drawSQL-export-2021-11-18_22_09

PS: Feel free to comment or provide feedback!

### CODE REVIEW

**Comment by carlomigueldy** [comment]
Oh I forgot to create a separate table for `profiles` @JoviDeCroock

**Comment by Dhaiwat10** [comment]
@carlomigueldy don't you think `gig_applications` should be linked with `users` (the applicant)?

File: database/schema.sql [hunk 1]
```diff
@@ -64,14 +64,15 @@ CREATE TABLE public.links (
...
 );
+COMMENT ON COLUMN public.links.created_by IS 'References to a wallet address from users table';
 
 
 
--- table: companies
-CREATE TABLE public.companies (
-  id uuid PRIMARY KEY NOT NULL DEFAULT gen_random_uuid(),
+-- table: organizations
+CREATE TABLE public.organizations (
```
Comment (JoviDeCroock) [inline database/schema.sql:74]: I love this rename

File: database/schema.sql [hunk 2]
```diff
@@ -107,36 +111,40 @@ CREATE TABLE public.jobs (
...
-  created_by uuid NOT NULL DEFAULT auth.uid()
+  created_by VARCHAR NOT NULL REFERENCES public.users (id)
 );
+COMMENT ON COLUMN public.job_applications.created_by IS 'References to a wallet address from users table';
 
 
 -- table: gigs
 CREATE TABLE public.gigs (
-  id uuid PRIMARY KEY NOT NULL DEFAULT gen_random_uuid(),
+  id UUID PRIMARY KEY NOT NULL DEFAULT gen_random_uuid(),
```
Comment (JoviDeCroock) [inline database/schema.sql:146]: Should we keep either a job or a gig? I don't know what the conclusion there was for the MVP

**Review by JoviDeCroock — APPROVED** [review]
I love this, we have org (dao/...) and a user (part of an org/someone who can take an org_gig)

**Review by angeljgomezc — APPROVED** [review]
Great!

**Comment by carlomigueldy** [comment]
Re: @Dhaiwat10 

> @carlomigueldy don't you think each gig_application should be linked with user (the applicant)? Same goes for job_applications

I kinda forgot to link those in the database design 🙈 , but from the database schema on file `schema.sql` the `job_applications` and `gig_applications` tables are linked with the `users` table by `created_by` attribute

File: database/schema.sql (hunk 1 above)
Comment (carlomigueldy) [reply database/schema.sql:74]: 💯

File: database/schema.sql (hunk 2 above)
Comment (carlomigueldy) [reply database/schema.sql:146]: I think we're keeping both, the other table could be empty for now when launching MVP

**Pull request merged**